packages/worker-proxy/src/index.ts

```diff
@@ -0,0 +1,115 @@
+import { createDeferred, Deferred } from '@trezor/utils/lib/createDeferred';
```

Comment:
creating some abstract proxy package makes sense to me. at the moment, we have 3 proxy situations that could be solved by this package:
1 - connect lives in service worker, caller lives in foreground
2 - transport with web bluetooth api lives in renderer/window, caller lives in node (suite-desktop case)
3 - connect lives in node, caller lives in renderer (suite-desktop case) 

can we come up with something abstract enough to handle all those cases and possible more cases that are yet to come?